packages/info-box/InfoBox.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+
+// import specific templates for the component
+import { XssSafeContent } from "@squiz/xaccel-xss-safe-content";
+import { Container } from "../grids/Container";
+import { LinkButton } from "../links/LinkButton";
+
+/**
+ * A container with a gray background that includes a
+ * title and text content with optional image and CTA button.
+ *
+ * @param {string} containerClassName
+ * Use this to apply classes to the box's outer container.
+ *
+ * @param {string} title
+ * A heading displayed at the top of the container
+ *
+ * @param {string} content
+ * WYSIWYG text content
+ *
+ * @param {object} imageData
+ * The image's data object of the selected Matrix asset. We need the URL and alt text.
+ *
```

Comment:
@iamrentman I also removed the prop` image` because we already have all we need from `imageData`, so image is no longer needed.